addons/point_of_sale/models/pos_order.py

```diff
@@ -842,6 +842,11 @@ def create(self, values):
         if not values.get('name'):
             # fallback on any pos.order sequence
             values['name'] = self.env['ir.sequence'].next_by_code('pos.order.line')
+
+        # Preventing warnings when pos_blackbox sends additional stuff
+        # (bool 'pro_forma_finalized' and char 'note')
```

Comment:
this part is not blocking, just making warning, so can be avoided in stable version to keep a minimal hack